0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -98,47 +100,46 @@ Ledger protocols are responsible for executing the individual transfers that con
 
 #### Minimal Support
 
-For minimal Interledger support, the ledger MUST have the ability for funds to be transferred from one account to another. This enables OTP transactions to pass through this ledger. It also allows third parties to act as escrow providers in order to enable UTP and ATP transactions on the ledger.
+For minimal Interledger support, the ledger MUST have the ability for funds to be transferred from one account to another. All of the more advanced ledger layer functionality, such as escrowed transfers, messaging, etc. can be implemented on top of this type of basic ledger.
 
 #### Basic Support
 
 For basic Interledger support, a ledger MUST fulfill the requirements for minimal support and also the following:
 
-The ledger MUST provide cryptographic escrow. Escrow means that transfers can have four states:
+The ledger MUST provide cryptographic authorization holds. During an authorization hold, money is put aside for a specific transfer until that transfer's outcome has been decided.
+
+Transfers using authorization holds can be in four distinct states:
 
-![enter image description here](https://lh3.googleusercontent.com/-QHsehrPTXgM/Vu_zs1dLWPI/AAAAAAAAEys/zchHKJj_MQs/s0/Transfer%252520States.png "Transfer States")
+![Transfers start in the proposed state, transition to the prepared state and finally to the executed state, which is final. Proposed and prepared transfers can also transition to the rejected state, which is final as well.](../shared/graphs/transfer-states.svg)
 
 * Proposed -- Nothing has happened yet.
```

Comment:
Agree that we can remove `Proposed`, but in its place we should add a name for the initial state, i.e. where the ledger does not know about the transfer yet.